src/App.js

```diff
@@ -14,10 +14,13 @@ function App() {
     <div className="App">
     <Header />
     <NavBar />
-    <Articles />
     <Routes>
-      <Route path='/articles' element={<ListArticles />} />
-      <Route path="/articles/:category_name" element={<Articles />} />
+      <Route path='/articles'>
+        <Route index element={<Articles topic="No Category" />} />
```

Comment:
This isn't how you should be doing this! This will make it dynamic:

`<Route path="/categories/:category_id" element={<Articles />} />`

Have a look in Articles.jsx for the thrilling conclusion...